v3/sockjs/session.go

```diff
@@ -30,7 +30,9 @@ var (
 	errSessionParse            = errors.New("sockjs: unable to parse URL for session")
 )
 
-type Session = *session
+type Session struct {
```

Comment:
@igm do I understand right that the goal here is to keep user code to use the same name `Session` as before without meed to change it to `*Session` throughout code base?

Reply:
yes, so that the existing code works without any modifications when upgrading to `v3`.